composer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "drupal/commerce_option",
+    "description": "Automates attribute creation.",
+    "type": "drupal-module",
+    "license": "GPL-2.0+",
+    "minimum-stability": "dev"
+}
```

Comment:
@thenchev We should require Drupal Commerce and add the authors as well to this file. Ie. https://github.com/shabananavas/drupal8-commerce-bluesnap/blob/8.x-1.x/composer.json

Reply:
Updated